src/integrationTest/java/org/opensearch/test/framework/matcher/AliasExistsMatcher.java

```diff
@@ -41,8 +41,8 @@ public AliasExistsMatcher(String aliasName) {
 	protected boolean matchesSafely(Client client, Description mismatchDescription) {
 		try {
 			GetAliasesResponse response = client.admin().indices().getAliases(new GetAliasesRequest(aliasName)).get();
-			ImmutableOpenMap<String, List<AliasMetadata>> aliases = response.getAliases();
-			Set<String> actualAliasNames = StreamSupport.stream(spliteratorUnknownSize(aliases.valuesIt(), IMMUTABLE), false)
+			Map<String, List<AliasMetadata>> aliases = response.getAliases();
```

Comment:
I think for this `getAliases()`, we also need to fix it, because it is also depending on the stale library of `ImmutableOpenMap`

Reply:
So I took a look into this and I think it will actually be resolved once the new build is done. For whatever reason, the build with the core change is still not complete: https://build.ci.opensearch.org/job/distribution-build-opensearch/7750/